src/heuristicblocking.js

```diff
@@ -181,9 +204,6 @@ HeuristicBlocker.prototype = {
     if(firstParties.indexOf(page_origin) === -1){
       firstParties.push(page_origin);
       snitch_map.setItem(tracker_origin, firstParties);
-      // TODO removing this line/messing up parameters should break integration tests
-      this.storage.setupHeuristicAction(tracker_fqdn, constants.ALLOW);
-      this.storage.setupHeuristicAction(tracker_origin, constants.ALLOW);
```

Comment:
I think these lines are still needed ... What they do is register the tracker's FQDN (`ssl.google-analytics.com`) and origin (`google-analytics.com`) in `action_map`, so that the user sees them (as unblocked) in the browser popup.